Gemfile

```diff
@@ -1,3 +1,4 @@
+# frozen_string_literal: true
```

Comment:
Unless you have a strong opinion about this, I'm going to leave these out of the PR. Not a huge fan of anything that forces you to sprinkle directives into every file.

If a user wants this functionality, they can instruct Ruby to do it at the VM-level with the `ruby ----enable=frozen-string-literal` flag, so hopefully that's an acceptable fallback?

Reply:
Don't feel strongly about it, just a habit from Shopify's coding practices carrying over 😄